src/index.js

```diff
@@ -0,0 +1,143 @@
+const cityNameInput = document.getElementById('cityNameInput');
+const cityNameResetBtn = document.getElementById('cityNameReset');
+const headerCityName = document.getElementById('headerCityName');
+const tempValue = document.getElementById('tempValue');
+const gardenEmoji = document.getElementById('gardenContent');
+const currentTempButton = document.getElementById('currentTempButton');
+const applyCityArrow = document.getElementById('applyCity');
+const skySelection = document.getElementById('skySelect');
+const skyDisplay = document.getElementById('skyDisplay');
+let isFahrenheit = true;
+let currentTemp = 50;
+
+function updateCityName() {
+  const city = cityNameInput.value.trim();
+  if (city) {
+    headerCityName.textContent = city;
+  } else {
+    headerCityName.textContent = '';
+  }
+}
+
+document.getElementById('toggleTempUnit').addEventListener('click', () => {
+  isFahrenheit = !isFahrenheit;
+  updateTempEmoji();
+  const button = document.getElementById('toggleTempUnit');
+  button.textContent = isFahrenheit ? '°C' : '°F';
+});
+
+applyCityArrow.addEventListener('click', () => {
+  updateCityName();
+});
+
+cityNameInput.addEventListener('change', updateCityName);
+const skyValue = skySelection.value;
+
+cityNameResetBtn.addEventListener('click', () => {
+  cityNameInput.value = 'Seattle';
+  headerCityName.textContent = 'Seattle';
+});
+
+const updateTempEmoji = () => {
+  const displayTemp = !isFahrenheit
+    ? currentTemp
+    : Math.round(((currentTemp - 32) * 5) / 9);
+  tempValue.textContent = `${displayTemp}`;
+
+  tempValue.classList.remove('red', 'orange', 'yellow', 'green');
```

Comment:
Typecasting your `displayTemp` into a string isn't necessary as it is done behind the scenes.
```suggestion
  const displayTemp = !isFahrenheit
    ? currentTemp
    : Math.round(((currentTemp - 32) * 5) / 9);
  
  tempValue.textContent = displayTemp;
  tempValue.classList.remove('red', 'orange', 'yellow', 'green');
```